transcom/mymove
# B-19681

## [B-19681](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A943857)
## [Integration Review](https://github.com/transcom/mymove/pull/12707)

## Summary

This task updates the 'About' screen when a Customer uploads ppm documents from zip code to full address. An issue was noticed during development due to the 'actualZipCode' being stored separately from the zip code originally entered by the user. The database currently does not have support for an actualAddress to be stored. After discussion with the customer it was decided that the actual zip code will still be saved separately, but the address field will just overwrite whatever the customer originally entered. This will be updated to support actualAddress in follow-on tasks. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Customer application
2. Login as a user 
3. Create a new move with a PPM and submit
4. Log out
5. Log in to the Office application
6. Login as a service counselor
7. Locate the move from step 3
8. Submit move details
9. Log out
10. Log in to the Customer application
11. Log in as the same user from step 2
12. Select the move from step 3
13. Click 'Upload PPM Documents'
14. Fill in the fields to include pickup and destination address (Use something different to validate that the update works)
15. Submit the form and complete the ppm
16. Log out
17. Access the Office Application
18. Login as a service counselor
19. Locate the move from step 3
20. View the PPM and verify that the address fields were updated.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
